vnext/Microsoft.ReactNative/ReactApplicationDelegate.cpp

```diff
@@ -20,10 +20,28 @@ using namespace Windows::ApplicationModel::Activation;
 
 namespace winrt::Microsoft::ReactNative::implementation {
 
-static void ApplyArguments(ReactNative::ReactNativeHost const & /*host*/, std::wstring const &arguments) noexcept {
-  // Microsoft::ReactNative::implementation::ReactNativeHost* hostImpl {
-  // get_self<Microsoft::ReactNative::implementation::ReactNativeHost>(host)};
+static void ApplyArguments(ReactNative::ReactNativeHost const &host, std::wstring const &arguments) noexcept {
+  Microsoft::ReactNative::implementation::ReactNativeHost *hostImpl{
+      get_self<Microsoft::ReactNative::implementation::ReactNativeHost>(host)};
   if (!arguments.empty() /*&& host.HasInstance()*/) {
+    constexpr wchar_t delimiter = L',';
```

Comment:
the comma seems like an odd delimiter to use here. can we use space? or colon : instead?

Reply:
I agree. I changed `WindowsStoreAppUtils.ps1` to use whitespace when starting the app.